### PR TITLE
feat: add histoire/nuxt component lib using nuxt subproject

### PR DESCRIPTION
**Pull request by connoratrug**
- add nuxt nuxt-components lib
- add histoire to  nuxt-components
- setup tailwind
- add workaround for tailwind relative path issue

how to test:
- explain here what to do to test this (or point to unit tests)

todo:
- [ ] updated docs in case of new feature
- [ ] added tests

### CODE REVIEW

**Comment by mswertz** [comment]
I like this direction. We should discuss how we gonna do this considering the whole molgenis suite. Minor decisions such as will we start merging all components into this (I suppose this one is tailwind only?) or will we have different apps for different component libs? And do I understand correctly that nuxt will cherry pick the code to be included so we could combine two paradigms: having multiple apps/libs (current situation) as well as direct code sharing between the apps without needing the intermediate build step (potentially not needing that damned yarn workspaces anymore?)

**Comment by mswertz** [comment]
ps, I got an error on preview 
image

**Comment by connoratrug** [comment]
> ps, I got an error on preview 


hmm i thought i had fixed that on local build by using your code from the histoire PR , i'm not that happy histoire config option , lots of  little hacks to host it as part of emx2 , ill try to fix it

**Comment by connoratrug** [comment]
> I like this direction. We should discuss how we gonna do this considering the whole molgenis suite. Minor decisions such as will we start merging all components into this (I suppose this one is tailwind only?)


yes its going to be hard to combine two css frameworks , if we decide to go with tailwind we maybe should consider moving the bootstrap components to tailwind ( nice job for llm :) ) 

 > or will we have different apps for different component libs? And do I understand correctly that nuxt will cherry pick the code to be included 

yes , but i need to really check this in the prod build, but thats my understanding 

> so we could combine two paradigms: having multiple apps/libs (current situation) as well as direct code sharing between the apps without needing the intermediate build step (potentially not needing that damned yarn workspaces anymore?)

when using vite as a dev server i suppose there no longer the need for multiple apps from a dev viewpoint, maybe there still is form a user viewpoint, but if you do ssr/really-good-tree-shaking that becomes more of a theoretical thing ( i.e browser should only load files need for current page/view).

File: apps/nuxt-components/stories/BackgroundGradient.story.vue [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<template>
+  <Story :layout="{ type: 'grid', width: '75%' }">
```
Comment (connoratrug) [inline apps/nuxt-components/stories/BackgroundGradient.story.vue:2]: i really don't understand how changing the layout solved the base url issue ,

Comment (mswertz) [reply apps/nuxt-components/stories/BackgroundGradient.story.vue:2]: it is a bug in histoire

**Comment by connoratrug** [comment]
need to wait for histoire people to fix / update there release , maybe then look at again

**Pull request closed without merging**